sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -7796,6 +7878,59 @@
               "input": true,
               "lockKey": true
             },
+            {
+              "title": "",
+              "customClass": "alert-banner-container alert-banner-warning",
+              "collapsible": false,
+              "hideLabel": true,
+              "key": "marriedCommonLawDomesticViolenceExceptionConfirmPanel3",
```

Comment:
Is the key meaningful for the content? It is only a panel and does not impact the functionality, but it may sidetrack or be confusing for maintenance in the future.

Reply:
Updated key to match confirmation content.